.travis.yml

```diff
@@ -2,16 +2,15 @@ language: dart
 
 dart:
   - 1.24.3
+  - 2.0.0
 
 # Re-use downloaded pub packages everywhere.
 cache:
   directories:
     - $HOME/.pub-cache
 
 script:
-- pub run dart_dev format --check
-- pub run dart_dev analyze
-- pub run dependency_validator -i coverage,dart_style,dartdoc
-- pub run dart_dev test
-- pub run dart_dev coverage --no-html
```

Comment:
note: coverage removed. it's probably possible to collect coverage without dart_dev since there are only vm tests. If people feel strongly about removing it, I could use help getting it working.